Remotion/Mixins/UnitTests/Core/CodeGeneration/AdHocCodeGenerator.cs

```diff
@@ -139,9 +135,6 @@ public string Save ()
         }
       }
 
-#if NETFRAMEWORK && ENABLE_PEVERIFY
```

Comment:
please leave a TODO comment
`TODO RM-9285: save generated assembly for use with IL verification.`